test/unit/test_sap_adt_datapreview.py

```diff
@@ -36,7 +36,7 @@ def test_parse_full(self):
                                     'MANDT': '000',
                                     'MTEXT': 'SAP SE',
                                     'MWAER': 'EUR',
-                                    'ORT01': 'Walldorf'},
+                                    'ORT01': 'Walldorf &amp; Brno &lt; London'},
```

Comment:
I do not think this is correct. We  should see `Walldorf & Brno < Lodon`. Why do you want to prevent XML parser to unescape?

Reply:
Please, see following comment with explanation.

Reply:
I still don't get it. Sorry.

Reply:
We are supposed to see `Walldorf & Brno < Lodon` and NOT  `Walldorf &amp; Brno &lt; London`.

Reply:
There in DB the string `Walldorf & Brno < Lodon` is defined. But, the ADT endpoint returns `Walldorf &amp; Brno &lt; London`. It is not encoded by the SAPCLI, the encoding of special characters is being done by ADT endpoint and returned to us

Reply:
I can try to apply `string.unescape` to see the expected result

Reply:
Done in commit cec84e8. `html.unescape(content)` is applied there. Now, you should see `Walldorf & Brno < London`